src/inject.js

```diff
@@ -44,12 +44,21 @@ const injectStyleTag = (cssContents /* : string */) => {
         }
     }
 
+    const sheet = ((styleTag.styleSheet || styleTag.sheet /* : any */) /* : CSSStyleSheet */);
 
-    if (styleTag.styleSheet) {
-        // $FlowFixMe: legacy Internet Explorer compatibility
-        styleTag.styleSheet.cssText += cssContents;
+    if (sheet.insertRule) {
+        cssContents.forEach((rule) => {
```

Comment:
Ran both versions through your benchmark several times. There was no noticeable difference between the `forEach` version and the `for` loop version, but a small speedup when storing `sheet.cssRules.length` in a variable.